### ISSUE TITLE
segfault when accessing output of [route]

### ISSUE THREAD

**Issue by tomara-x** [plugdata-team/plugdata#477]
https://user-images.githubusercontent.com/86204514/216553700-baf497e4-5eff-45cc-9a5e-f9de40b6c78d.mp4

connecting any output of [route] to either a numbox, [display], or [print] will cause a crash (that's what i tested at least)
even a patch like:
```
#X obj 279 176 route hi;
#X obj 279 129 nbx 4 18 -1e+37 1e+37 0 0 empty empty empty 0 -8 0 10 #191919 #ffffff #ffffff 0 256;
#X obj 312 224 nbx 4 18 -1e+37 1e+37 0 0 empty empty empty 0 -8 0 10 #191919 #ffffff #ffffff 0 256;
#X connect 0 1 2 0 empty;
#X connect 1 0 0 0 empty;
```
found wile trying to understand why the numboxes in hv.dispatch's help file won't update, while they work fine in vanilla

**Comment by tomara-x** [comment]
not the same but i think it might be related:

https://user-images.githubusercontent.com/86204514/216813025-998610b1-9f08-44b3-bba0-70058511fce6.mp4

**Comment by timothyschoen** [comment]
Good that you caught this! Apparently, pd can sometimes send messages where the method symbol is a nullptr, I didn't count on it that that would ever happen. Should be fixed with 73af93c!

**Comment by timothyschoen** [comment]
The other one you posted also appears to be fixed!
